app/src/main/res/values/strings.xml

```diff
@@ -221,6 +221,10 @@
     <string name="nc_call_state_with_phone">%1$s with phone</string>
     <string name="nc_call_state_with_video">%1$s with video</string>
 
+    <!-- Picture in Picture -->
+    <string name="nc_pip_microphone_mute">mute microphone</string>
+    <string name="nc_pip_microphone_unmute">enable microphone</string>
```

Comment:
I would guess we do sentence casing here, so
`Mute microphone`
`Enable microphone`